src/main/java/yapbot/main/YapBot.java

```diff
@@ -0,0 +1,566 @@
+package yapbot.main;
+
+import yapbot.tasks.Deadline;
+import yapbot.tasks.Event;
+import yapbot.tasks.Task;
+import yapbot.tasks.ToDo;
+import yapbot.exceptions.YapBotException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Scanner;
+
+/**
+ * Main class to start YapBot.
+ */
+public class YapBot {
+    private static final String PREFIX_LINE = "\n-------------------------------------------";
+    private static final String POSTFIX_LINE = "-------------------------------------------\n";
```

Comment:
the prefix and postfix line could maybe defined in another interface since it is used often?